src/openrct2/interface/Window_internal.h

```diff
@@ -62,6 +62,7 @@ struct rct_window
         int16_t picked_peep_old_x; // staff/guest window: peep x gets set to 0x8000 on pickup, this is the old value
         int16_t vehicleIndex;      // Ride window: selected car when setting vehicle colours
         int16_t numberOfStaff;     // Used in park window.
+        int16_t SceneryEntry;      // Used in sign window.
         int16_t var_48C;
```

Comment:
Should this be gone? Or is it being used in other places where it doesn't mean `SceneryEntry`?

Reply:
I don't understand, could you elaborate?

Reply:
```c++
    w->var_48C = tile_element->AsLargeScenery()->GetEntryIndex();
```
became
```
    w->SceneryEntry = tile_element->AsLargeScenery()->GetEntryIndex();
```

Which is a new variable, instead of renaming `var_48C`, my question is if `var_48C` is used in other places for different purposes.

Reply:
Ah, I see. I'd say it could be useful as an indication that this is a union. The alternative is making the other windows responsible for initialising their own variable (so the sign window initialises SceneryEntry, the park windows initialises numberOfStaff, etc.).